logger.py

```diff
@@ -0,0 +1,14 @@
+import os
+import logging
+
+LOG_LEVEL = logging.DEBUG if not os.getenv('NO_DEBUG_MODE') else logging.INFO
+LOG_FORMAT = '[%(asctime)s] %(filename)s %(funcName)s %(lineno)d <%(levelname)s>: %(message)s'
+logging.basicConfig(format=LOG_FORMAT)
+
+app_logger = logging.getLogger('app')
+log_queue_logger = logging.getLogger('querylog')
+jsonbin_logger = logging.getLogger('jsonbin')
+hedy_content_logger = logging.getLogger('hedy_content')
+
+for _logger in (app_logger, log_queue_logger, jsonbin_logger):
+    _logger.setLevel(LOG_LEVEL)
```

Comment:
I believe that you can also configure a level as part of calling `logging.basicConfig()`. Does that remove the need for configuring the log level on each individual logger?

------

And if that removes the need for configuring individual loggers, then do we still need to centralize all loggers here in this one file? I was under the impression it's pretty common to initialize a logger in every file, using the current file name.

I think you're even supposed to do this:

```py
logger = logging.getLogger(__name__)
```

In every file. At least, that's what [this page](https://docs.python.org/3/library/logging.html#logger-objects) tells me. 🤔

Reply:
This was my initial approach, but surprisingly it didn't produce the expected results. I had to add the explicit `setLevel` call at the logger level to see the logs being issued. There seem to be a side effect when having `import logging` in `website/log_queue.py`, the configuration gets overwritten.